src/main/java/uk/gov/hmcts/reform/civil/service/PaymentsService.java

```diff
@@ -36,18 +37,35 @@ private CreditAccountPaymentRequest buildRequest(CaseData caseData) {
         String customerReference = ofNullable(caseData.getClaimIssuedPaymentDetails())
             .map(PaymentDetails::getCustomerReference)
             .orElse(caseData.getPaymentReference());
+        CreditAccountPaymentRequest creditAccountPaymentRequest = null;
 
-        return CreditAccountPaymentRequest.builder()
-            .accountNumber(caseData.getApplicantSolicitor1PbaAccounts().getValue().getLabel())
-            .amount(claimFee.getCalculatedAmount())
-            .caseReference(caseData.getLegacyCaseReference())
-            .ccdCaseNumber(caseData.getCcdCaseReference().toString())
-            .customerReference(customerReference)
-            .description("Claim issue payment")
-            .organisationName(organisationName)
-            .service(paymentsConfiguration.getService())
-            .siteId(paymentsConfiguration.getSiteId())
-            .fees(new FeeDto[]{claimFee})
-            .build();
+        if (!SPEC_CLAIM.equals(caseData.getSuperClaimType()))  {
+            creditAccountPaymentRequest = CreditAccountPaymentRequest.builder()
+                .accountNumber(caseData.getApplicantSolicitor1PbaAccounts().getValue().getLabel())
+                .amount(claimFee.getCalculatedAmount())
+                .caseReference(caseData.getLegacyCaseReference())
+                .ccdCaseNumber(caseData.getCcdCaseReference().toString())
+                .customerReference(customerReference)
+                .description("Claim issue payment")
+                .organisationName(organisationName)
+                .service(paymentsConfiguration.getService())
+                .siteId(paymentsConfiguration.getSiteId())
+                .fees(new FeeDto[]{claimFee})
+                .build();
+        } else if (SPEC_CLAIM.equals(caseData.getSuperClaimType())) {
+            creditAccountPaymentRequest = CreditAccountPaymentRequest.builder()
+                .accountNumber(caseData.getApplicantSolicitor1PbaAccounts().getValue().getLabel())
+                .amount(claimFee.getCalculatedAmount())
+                .caseReference(caseData.getLegacyCaseReference())
+                .ccdCaseNumber(caseData.getCcdCaseReference().toString())
+                .customerReference(customerReference)
+                .description("Claim issue payment")
+                .organisationName(organisationName)
+                .service(paymentsConfiguration.getSpecService())
```

Comment:
since service and siteId are the only difference between spec and unspec, I think it'd be easier to see that if the if blocks contained only that, something like
builder = CreditAccountPaymentRequest.builder().<...commonProperties...>();
if (SPEC_CLAIM) {
  builder.service(specService).siteId(specSiteId);
} else {
  builder.service(service).siteId(siteId);
}
return builder.build();